src/renderer/pkijs/tests/verification.ts

```diff
@@ -0,0 +1,11 @@
+import { getCrypto, getAlgorithmParameters } from 'pkijs'
+
+import { signAlg } from '../generatePems/config'
+
+export const verificationSignature = async (userPubKey, signature, message) => {
```

Comment:
typo: "verifySignature"

also please add return type (Promise<boolean>) and parameter types